modules/switch/react/lib/Switch.tsx

```diff
@@ -126,7 +126,9 @@ const SwitchBackground = styled('div')<Pick<SwitchProps, 'checked' | 'disabled'>
   ({checked, disabled, theme}) => {
```

Comment:
is there a reason of not using destructuring from theme? 
```
({checked, disabled, theme: {canvas:{palette}}}) => {
 backgroundColor: paletter.primary.light
 }
```

Reply:
Just pushed an update with destructuring. I couldn't do it everywhere because some of our helper functions rely on the `theme` object. Was able to update it in most places though! 👍